pkgs/development/libraries/opencolorio/default.nix

```diff
@@ -25,8 +25,8 @@ stdenv.mkDerivation rec {
     sha256 = "194j9jp5c8ws0fryiz936wyinphnpzwpqnzvw9ryx6rbiwrba487";
   };
 
-  nativeBuildInputs = [ cmake ];
-  buildInputs = [ expat libyamlcpp ilmbase pystring ]
+  nativeBuildInputs = [ cmake (symlinkJoin { name = "expat"; paths = [ expat.out expat.dev ]; }) ];
```

Comment:
Why isn't expat.out enough? It should contain all outputs.

Reply:
it would lack the expat cmake files, returning something like this
```
CMake Error at /nix/store/d57wa2g6c84fyskymws68qr2633a2hc1-cmake-3.19.7/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:218 (message):
  Could NOT find expat (missing: expat_INCLUDE_DIR) (Required is at least
  version "2.2.8")
Call Stack (most recent call first):
  /nix/store/d57wa2g6c84fyskymws68qr2633a2hc1-cmake-3.19.7/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:582 (_FPHSA_FAILURE_MESSAGE)
  share/cmake/modules/Findexpat.cmake:113 (find_package_handle_standard_args)
  share/cmake/modules/FindExtPackages.cmake:29 (find_package)
  CMakeLists.txt:239 (include)
```

if you remove `expat.out`, it would lack the expat shared libraries

Reply:
paging @SuperSandro2000